chia/types/generator_types.py

```diff
@@ -37,3 +37,5 @@ class NewBlockGenerator(BlockGenerator):
     additions: list[Coin] = field(default_factory=list)
     # all coins being spent by the block generator
     removals: list[Coin] = field(default_factory=list)
+    # the total cost of the block generator, CLVM + bytes + conditions
+    cost: uint64 = field(default=uint64(0))
```

Comment:
The cost field is added to the `NewBlockGenerator` class, which is the output from `create_block_generator()` and `create_block_generator2()`